src/main/MQ2Benchmarks.cpp

```diff
@@ -102,33 +102,53 @@ bool GetMQ2Benchmark(uint32_t BMHandle, MQBenchmark& Dest)
 
 void Cmd_DumpBenchmarks(SPAWNINFO* pChar, char* szLine)
 {
+	// Execute and time a command starting with '/'
 	if (szLine && szLine[0] == '/')
 	{
-		uint64_t Start = MQGetTickCount64();
+		uint64_t start = MQGetTickCount64();
 		DoCommand(szLine, false);
+		double time = (MQGetTickCount64() - start) / 1000.0;
+		WriteChatf("\ay%s\ax took \at%.2f\axs", szLine, time);
+		return;
+	}
 
-		uint64_t Time = MQGetTickCount64() - Start;
-		WriteChatf("\ay%s\ax completed in \at%.2f\axs", szLine, static_cast<double>(Time) / 1000.);
+	// Since it doesn't start with a slash, let's check there is a benchmark name to match
+	// "/benchmark mq2nav" for example
+	if (szLine && szLine[0])
+	{
+		WriteChatf("Start %s Benchmark", szLine);
+		WriteChatColor("--------------");
```

Comment:
This should be moved into the `if` clause below, such that it doesn't print if nothing is found.

If nothing is found, it should print that at the end, too.